otj-server-mvc/src/main/java/com/opentable/server/mvc/ExceptionLogFilter.java

```diff
@@ -0,0 +1,54 @@
+package com.opentable.server.mvc;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+import org.eclipse.jetty.io.QuietException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.NestedServletException;
+
+
+@Component
+public class ExceptionLogFilter implements Filter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ExceptionLogFilter.class);
+
+    @Override
+    public void init(FilterConfig filterConfig) {
+        LOG.info("Exception log filter enabled.");
+    }
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
```

Comment:
What's the point of this? Won't the framework log on its own? Why do we need to wrap the exception?